docs/user/dashboard/url-drilldown.asciidoc

```diff
@@ -240,7 +150,7 @@ For example, `{{context.panel.filters}}` are previewed with the current filters
 *Event* variables are extracted during drilldown execution from a user interaction with a panel (for example, from a pie slice that the user clicked on).
 
 Because there is no user interaction with a panel in preview, there is no interaction context to use in a preview.
-To work around this, {kib} provides a sample interaction that relies on a picked <<trigger-picker, trigger>>.
+To work around this, {kib} provides a sample interaction that relies on a trigger.
 So in a preview, you might notice that `{{event.value}}` is replaced with `{{event.value}}` instead of with a sample from your data.
```

Comment:
`{{event.value}}` is replaced with `{{event.value}}`

These two look the same. Are they supposed to reference different variable?

Reply:
@Dosant

Reply:
This section explains that we don't have a sample data from a chart to make a proper replacement of `{{event.*}}` in preview. So in preview `{{event.value}}` variable is literally propagated as is: `{{event.value}}` -> `{{event.value}}`. 
We will revise this limitation in future releases. Not sure how to explain this clearer in docs now..